bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -578,114 +457,77 @@ All these channels except welcomePersonAtHome are read only.
-DateTime             Indoor_Max_Noise_This_Week_TS     "Max Noise This Week [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"        <calendar>         { channel = "netatmo:NAMain:home:inside:DateMaxNoiseThisWeek" }
-DateTime             Indoor_Max_Noise_This_Month_TS    "Max Noise This Month [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"       <calendar>         { channel = "netatmo:NAMain:home:inside:DateMaxNoiseThisMonth" }
-Number               Indoor_WifiStatus                 "WifiStatus [%s]"                                            <signal>           { channel = "netatmo:NAMain:home:inside:WifiStatus" }
-DateTime             Indoor_TimeStamp                  "TimeStamp [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                  <calendar>         { channel = "netatmo:NAMain:home:inside:TimeStamp" }
-Location             Indoor_Location                   "Location"                                                   <movecontrol>      { channel = "netatmo:NAMain:home:inside:Location" }
-DateTime             Indoor_LastStatusStore            "LastStatusStore [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"            <text>             { channel = "netatmo:NAMain:home:inside:LastStatusStore" }
+Number               Indoor_RadioStatus                 "RadioStatus [%s]"                                            <signal>           { channel = "netatmo:NAMain:home:inside:RadioStatus" }
+DateTime             Indoor_TimeStamp                  "TimeStamp [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                  <calendar>         { channel = "netatmo:NAMain:home:inside:timestamp" }
+DateTime             Indoor_LastSeen            "LastSeen [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"            <text>             { channel = "netatmo:NAMain:home:inside:last-seen" }
 
 # Outdoor Module
-Number:Temperature   Outdoor_Temperature               "Temperature [%.1f %unit%]"                                  <temperature>      { channel = "netatmo:NAModule1:home:outside:Temperature" }
-String               Outdoor_TempTrend                 "TempTrend [%s]"                                             <line>             { channel = "netatmo:NAModule1:home:outside:TempTrend" }
-Number:Dimensionless Outdoor_Humidity                  "Humidity [%d %unit%]"                                       <humidity>         { channel = "netatmo:NAModule1:home:outside:Humidity" }
-Number               Outdoor_Humidex                   "Humidex [%.0f]"                                             <temperature_hot>  { channel = "netatmo:NAModule1:home:outside:Humidex" }
-Number:Temperature   Outdoor_HeatIndex                 "HeatIndex [%.1f %unit%]"                                    <temperature_hot>  { channel = "netatmo:NAModule1:home:outside:HeatIndex" }
+Number:Temperature   Outdoor_Temperature               "Temperature [%.1f %unit%]"                                  <temperature>      { channel = "netatmo:NAModule1:home:outside:temperature" }
```

Comment:
I believe it's now:
```
netatmo:NAModule1:home:inside:outside:temperature#value
                       ^^^^^^^                   ^^^^^^
```

The same is true for many other items, I'll save us the trouble of mentioning each one for now. :)